src/storage-client/src/types/parameters.proto

```diff
@@ -10,14 +10,10 @@
 syntax = "proto3";
 
 import "proto/src/proto.proto";
+import "persist-client/src/cfg.proto";
 
 package mz_storage_client.types.parameters;
 
 message ProtoStorageParameters {
-    ProtoPersistParameters persist = 1;
```

Comment:
Do we have tests that validate that moving types to a different namespace is backwards compatible?